store/sqlstore/plugin_store.go

```diff
@@ -12,6 +12,10 @@ import (
 	"github.com/mattermost/mattermost-server/store"
 )
 
+const (
+	DEFAULT_KEY_FETCH_LIMIT = 10
```

Comment:
Non-blocking but might be worth throwing the word `PLUGIN` in there somewhere as this will be usable across the whole package